mobile/lib/features/wallet/wallet_screen.dart

```diff
@@ -138,10 +142,30 @@ class _WalletScreenState extends State<WalletScreen> {
                                         fontStyle: FontStyle.italic,
                                       ),
                                     )
-                                  : AmountText(
-                                      amount: walletChangeNotifier.total(),
-                                      textStyle: const TextStyle(
-                                          fontSize: 20.0, fontWeight: FontWeight.bold))),
+                                  : Row(
+                                      children: [
+                                        AmountText(
+                                            amount: walletChangeNotifier.total(),
+                                            textStyle: const TextStyle(
+                                                fontSize: 20.0, fontWeight: FontWeight.bold)),
+                                        Visibility(
+                                            visible:
+                                                positionChangeNotifier.getStableSatsAmount() != 0.0,
+                                            child: Row(
+                                              children: [
+                                                const SizedBox(width: 5),
+                                                const Text("+"),
```

Comment:
🔧 Combining the balances of sats and $ is very irritating for me. IMHO this balance should only ever show one value. In other wallets (e.g. blink or zeus) they let you select what currency you prefer and show you the balance correspondingly. 

Equally, I would like to see our balance to do the same. If we want to show the synthetic usd balance we should simply add it on the balance in the currency selected. btc, usd, eur, etc.

Reply:
can we try and see whether people prefer it? in the spirit of experiment? 

it's not a simple UX matter, because other wallets you mentioned do not have the synthetic stablecoin version. 
Please note that Galloy in their stablesats put exactly this approach (although in a nicer UI, to be honest!)


the idea was: 
- make it always visible that user

> Equally, I would like to see our balance to do the same. If we want to show the synthetic usd balance we should simply add it on the balance in the currency selected. btc, usd, eur, etc.

this hides the whole stablesats feature and entirely misses the point. 


The discussion whether to display currencies is close to me, and I think we should do that separately anyway, if we want to be a *good* lightning wallet.

Reply:
Personally I don't really mind either way. The benefit of displaying sats + dollars is ease of view - stabilised sats seem to feel more "first class" this way.

Reply:
> it's not a simple UX matter, because other wallets you mentioned do not have the synthetic stablecoin version.

Have you had a look at blink? It's the galoy stablesats wallet.

Reply:
> > it's not a simple UX matter, because other wallets you mentioned do not have the synthetic stablecoin version.
> 
> Have you had a look at blink? It's the galoy stablesats wallet.

yup, they have both always visible by default *and* the feature of showing everything in USD (which we don't). I don't have an Android, so I can't really test it out.

Reply:
I am not against showing the Synthetic usd balance, it's just the way it is shown with your change seems inconsistent. 

I see the following issues

- We don't show a balance for the long positions (As you have pointed out yourself)
- We sometimes show a balance with multiple currencies. Contrary to your comment, blink does not do the same, it shows multiple accounts which we do as well with the difference that we collapse them, but it does not show two different balances in on row.
- The sats for the synthetic usd is still shown as an outgoing transaction.

I really would prefer to go back to Figma with that and think it through before releasing this change as it's not critical for the experiment and tackles a component that needs a redesign anyways.

Reply:
If you want to bring it in quickly and can't live with that part of the PR, just change the Visibility widget to 'false'. 
I think we'd have exactly the same conversation in Figma, I can't imagine that changing anything- happy to split it , or perhaps *iterate* on it seeing feedback for customers.

Reply:
> If you want to bring it in quickly and can't live with that part of the PR, just change the Visibility widget to 'false'.

what do you mean by that?

> I think we'd have exactly the same conversation in Figma, I can't imagine that changing anything- happy to split it , or perhaps iterate on it seeing feedback for customers.

I think we will have a different discussion there, because we can think about the bigger picture, including how this should look like in the history, if we should stick with a collapsible balance, how we should show long positions (e.g. keep them in the balance), show a balance in a single currency, etc.

Reply:
> what do you mean by that?

Sorry, I was on my phone, so tried to reply briefly before. I meant changing the `Visibility` widget's `visible` property to `false`, because this is the sticking point for this issue.

Are you actively against this PR? I don't mind iterating on this further at all, because I don't think we've nailed it, but I certainly think this is step in the right direction. 

I would like to get some clarification from you before I go ahead and merge it. Right now it's in an awkward limbo state.

PS the lack of Figma / big picture discussion also applies to the whole stablesats feature, not just this part of it. It's a quick experiment / hack to gather reactions and early feedback.

Reply:
> Are you actively against this PR? I don't mind iterating on this further at all, because I don't think we've nailed it, but I certainly think this is step in the right direction.

I am only against the part where we show the balance with "45100 sats + 10$". I am fine with the rest though. But that's just my opinion. I am not going to block that PR, because of it.

Reply:
noted, thank you for the feedback. 
let's give it a spin towards until the end of the week, and I'm happy to remove it if it is weird after that period of internal testing, ok? 
let's see how that feels on the device.